data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/mapping/LogstashPluginMapper.java

```diff
@@ -50,14 +49,19 @@ public PluginModel mapPlugin(LogstashPlugin logstashPlugin) {
             throw new LogstashMappingException("Unable to parse mapping file " + mappingResourceName, ex);
         }
 
+        if(logstashMappingModel.getCustomPluginMapperClass() != null) {
+            final LogstashPluginAttributesMapper mapper = pluginMapperProvider.getAttributesMapper(logstashMappingModel);
+            return mapper.mapAttributes(logstashPlugin.getAttributes(), logstashMappingModel);
+        }
```

Comment:
I don't think we need this conditional statement. The remaining lines of the function are duplicate of the conditional statement body.

Reply:
We will run into the error of plugin name not being defined when checked later down the code if this conditional does not exist